lib/dotenv-flow.js

```diff
@@ -64,6 +62,7 @@ function getDotenvFilenames(cwd) {
  * @param {string} [options.cwd=process.cwd()] – path to `.env*` files directory
  * @param {string} [options.encoding=utf8] – encoding of `.env*` files
  * @param {boolean} [options.purge_dotenv] – perform the {@link cleanupDotenvDefinedVars}
+ * @param {string} [options.node_env=process.env.NODE_ENV] – custom environment name
```

Comment:
Yeah, like this term 👍

```suggestion
 * @param {string} [options.node_env=process.env.NODE_ENV] – custom environment target
```